src/groups/mqb/mqbcmd/mqbcmd_parseutil.cpp

```diff
@@ -223,40 +223,64 @@ int parseClearCache(ClearCache*              command,
                     WordGenerator            next,
                     const bslstl::StringRef& commandPrefix);
 
+int parseEncodingFormat(EncodingFormat::Value* format,
+                        bsl::string*           error,
+                        WordGenerator&         next);
```

Comment:
Might be worth it to refactor everything in this file to pass all args as references